app/controllers/page_views_controller.rb

```diff
@@ -4,74 +4,31 @@ class PageViewsController < ApplicationMetalController
   include ActionController::Head
 
   def create
-    page_view_create_params = if session_current_user_id
-                                page_view_params.merge(user_id: session_current_user_id)
-                              else
-                                page_view_params.merge(counts_for_number_of_views: 10)
-                              end
-
-    PageView.create(page_view_create_params)
+    page_view_create_params = params.slice(:article_id, :referrer, :user_agent)
+    if session_current_user_id
+      page_view_create_params[:user_id] = session_current_user_id
+    else
+      page_view_create_params[:counts_for_number_of_views] = 10
```

Comment:
Does anyone know where the number 10 comes from here?

Reply:
It comes from #1985

IIRC it was an arbitrary number to have a few views for each article

Reply:
Thanks, @rhymes. With this new method of updating, does this still make sense or should we use 1 instead?

Reply:
I think it'd make sense now that we're updating all the time. I'm wondering if it's going to skew any internal statistic that we're keeping at Forem? Aside from internal dashboards, IIRC we use page views for the `AnalyticsService` class and the API using it. Last but not least, we need to check if there's anywhere we display articles by page views that might alter the number short term

Reply:
@benhalpern You probably have some knowledge around this.

Reply:
The number ten matches up with the frequency with which we call this on the client. On the client we only call this action one in ten times, so on the server, each time we call it it "counts for" ten visits with non-authenticated traffic.

Reply:
Context froma  Slack conversation with @benhalpern:

```js
    // page view
    if (ArticleElement && tokenMeta && !isBot) {
      var randomNumber = Math.floor(Math.random() * 10); // 1 in 10; Only track 1 in 10 impressions
      if (!checkUserLoggedIn() && randomNumber != 1) {
        return;
      }
```

> Yes, we should keep it at ten. Technically if we changed it in both places, we could change it, but that should probably be a bigger conversation.

Caveat: 

> // 1 in 10; Only track 1 in 10 impressions

This isn't really 1 in 10 impressions, it's a 1 in 10 chance each time this code runs. It could be true 10 times in a row or false 100 times in a row, unlikely as that may be.